jobs/pacman-rule-engine-2.0/src/main/java/com/tmobile/pacman/publisher/impl/AnnotationPublisher.java

```diff
@@ -486,9 +483,8 @@ private String buildIssueKey(Map<String, String> resource) {
      *
      * @param responseMapList the response map list
      */
-    private void processErrors(List<Map<String, Map>> responseMapList) {
-        logger.error("some errors occured while publishing the anotation, but no error handler found to handle it",
-                responseMapList);
+    private void processErrors(List<Map<String, Map>> responseMapList, String response) {
+        logger.error("Error occurred in job while publishing the annotation, but no error handler found to handle it. Number of errors is "+responseMapList.size()+" "+ response);

```

Comment:
This about showing contents of the responseMapList in the log. Please discuss with @santhosh-challa